src/SFA.DAS.RoATPService.Importer/Validators/RegisterEntryValidator.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace SFA.DAS.RoATPService.Importer
+{
+    using SFA.DAS.RoATPService.Importer.Models;
+    using System;
+    using System.Collections.Generic;
+
+    public class RegisterEntryValidator
+    {
+        public RegisterEntryValidationResult ValidateRegisterEntry(RegisterEntry registerEntry)
+        {
+            var validationResult = new RegisterEntryValidationResult();
+
+            var errorMessages = new List<string>();
+
+            if (registerEntry.ProviderTypeId < 1 || registerEntry.ProviderTypeId > 3)
+            {
+                errorMessages.Add("Application route id should be between 1 and 3");
+            }
+
+            if (registerEntry.UKPRN < 10000000 || registerEntry.UKPRN > 99999999)
+            {
+                errorMessages.Add("UKPRN should be between 10000000 and 99999999");
+            }
+
+            if (registerEntry.OrganisationTypeId != 0)
```

Comment:
Is this sufficient validation?

Reply:
At the moment, yes. We only have an organisation type of '0 - Unassigned'
When the list of organisation types has been ratified and added to the lookup data, this will need to be updated.